src/components/KTreeList/KTreeItem.vue

```diff
@@ -1,41 +1,64 @@
 <template>
-  <button
-    class="tree-item"
-    :class="{
-      'not-draggable': disabled,
-      'selected': item.selected
-    }"
-    :data-testid="`tree-item-${item.id}`"
-    data-tree-item-trigger="true"
-    draggable="false"
-    type="button"
-    @click.prevent="handleClick"
+  <div
+    class="tree-item-wrapper"
+    :data-testid="`tree-item-wrapper-${item.id}`"
   >
-    <div
-      v-if="hasIcon"
-      class="tree-item-icon"
-      data-testid="tree-item-icon"
+    <ChevronDownIcon
```

Comment:
A few issues:

1. The chevron should point "down" when expanded, and "right" when collapsed.
2. Clicking the button seems to toggle the icon direction.
3. After reordering the items, but not touching collapse icon button, the icon direction seems to change on the docs site.
4. After reordering, I can no longer expand/collapse the items on the first click consistently. Sometimes it takes several clicks.